tests/cypress/e2e/Teacher page/Manual/FrequentMistakesLevels_button.cy.js

```diff
@@ -0,0 +1,44 @@
+import {loginForAdmin, loginForTeacher} from '../../tools/login/login.js'
+import { goToHome, goToLogin, goToRegister, goToPage } from "../../tools/navigation/nav.js";
+
+describe('Is able to see teacher page', () => {
+  it('Passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    goToPage('/for-teachers/manual');
+    cy.get('#button-4').click();
+
+    for(let i = 1; i <= 18; i++)
+    {
+        cy.get('[onclick="$(\'#common_mistakes-' + i +  '\').slideToggle();"]').click();
+    }
```

Comment:
Isn't it possible to get this dynamically? Now we hard-code that there are 18 buttons; that is not what we want to test. We want to test if, for every button there is, it works. Some languages might not be up-to-date with the English/Dutch translations and miss a few. In this case the tests would fail but the fuctionality is fine.